src/garn-bin.js

```diff
@@ -92,6 +92,7 @@ function compile() {
   parsed.options.module = ts.ModuleKind.CommonJS;
   parsed.options.jsx = ts.JsxEmit.React;
   parsed.options.incremental = true;
+  parsed.options.rootDir = backSlashToForwardSlash(path.relative(buildsystemPath, basePath));
```

Comment:
Setting the rootDir to the workspace's  basePath won't allow us to import files from eg. the project's root buildsystem, like we do here: https://github.com/avensia/excite-packages/blob/master/packages/core/buildsystem/index.ts#L2

However, setting the rootDir to the actual project's root directory will allow us to include any file that exists across the project. 
```suggestion
  parsed.options.rootDir = backSlashToForwardSlash(rootPath);
```

Reply:
Good catch!